src/Meta/Alert_Meta.php

```diff
@@ -207,6 +222,21 @@ private function get_rules_group(): ACF\Field_Group {
 		return $group;
 	}
 
+	private function get_colors_field(): ACF\Field {
+		$field = new ACF\Field( self::NAME . '_' . self::FIELD_COLOR );
+
+		$field->set_attributes( [
+			'label'         => __( 'Backgroung Color', 'tribe-alerts' ),
+			'name'          => self::FIELD_COLOR,
+			'type'          => 'radio',
```

Comment:
This is not the correct field type, we brought in the acf-swatch plugin for that specific field type and that's what we should be using here.